### DISCUSSION TITLE
Video player accessibility

### DISCUSSION THREAD

**Discussion by jweintraub** [twilio-labs/paste#1688] [Q&A] [answered]
Does Paste have specific guidance of use of embedded videos and which video player to use? 
My use case is for the Console overview template which will have an optional section for an embedded video.

**Comment by TheSisb** [comment, marked as answer]
Hi @jweintraub,

We haven't yet worked on guidance for video players. What you will want to be mindful of is meeting A and AA WCAG guidelines for video. This two documents should help: https://www.w3.org/WAI/media/av/planning/#checklist and https://www.w3.org/WAI/media/av/av-content/#video. AAA would be a nice to have, but not a blocker.

We'd really appreciate it if you could report back what you end up going with, so we can keep that documented for when we work on this in the future.

**Reply by aayushpi** [reply]
We have some guidance around alternative text in our Inclusive Design Guide for mixed media content. 

In general, pair text for users and situations where video is not available for consumption, and provide captioning for all video resources.


https://paste.twilio.design/inclusive-design#general-considerations
